app/components/provider_interface/interview_details_component.rb

```diff
@@ -8,29 +8,72 @@ def initialize(interview_form, interview = nil)
     end
 
     def rows
-      date_row = build_row('Date', @interview_form.date_and_time.to_s(:govuk_date))
-      time_row = build_row('Time', @interview_form.date_and_time.to_s(:govuk_time))
-      organisation_row = build_row('Organisation carrying out interview', @interview_form.provider.name)
-      location_row = build_row('Address or online meeting details', @interview_form.location)
-      details_row = build_row('Additional details', @interview_form.additional_details.presence || 'None')
+      [
+        date_row,
+        time_row,
+        organisation_row,
+        location_row,
+        details_row,
+      ]
+    end
+
+  private
+
+    def date_row
+      build_row(:date, @interview_form.date_and_time.to_s(:govuk_date))
+    end
+
+    def time_row
+      build_row(:time, @interview_form.date_and_time.to_s(:govuk_time))
+    end
+
+    def organisation_row
+      build_row(:provider_id, @interview_form.provider.name)
+    end
+
+    def location_row
+      build_row(:location, @interview_form.location)
+    end
 
-      [date_row, time_row, organisation_row, location_row, details_row]
+    def details_row
+      build_row(:additional_details, @interview_form.additional_details.presence || 'None')
     end
 
-    def build_row(key, value)
+    def build_row(field, value)
+      key = key_for_field(field)
       {
         key: key,
         value: value,
-        action: "Change #{key.downcase}",
-        change_path: change_path,
+        action: key.downcase,
+        change_path: change_path(field),
       }
     end
 
-    def change_path
+    def key_for_field(field)
+      if %i[date provider_id].include?(field)
```

Comment:
We will need to include the provider id in the anchor as the form fields include those ids, example:
```
<input id="provider-interface-interview-wizard-provider-id-319-field" class="govuk-radios__input" type="radio" value="319" checked="checked" name="provider_interface_interview_wizard[provider_id]">
```

Reply:
Good spot!

Reply:
Ah I actually noticed that the error messages as well don't have the correct anchor to this section, should we tackle that in  this PR or add a new ticket?
![Screenshot 2021-02-16 at 14 11 32](https://user-images.githubusercontent.com/2732945/108061391-e836aa00-7060-11eb-97f7-0fbe94238114.png)

Reply:
Easy to fix the latter issue – I'll do it here.